### ISSUE TITLE
Finetuning CUT: load_size ignored

### ISSUE THREAD

**Issue by Hirnmoder** [taesungp/contrastive-unpaired-translation#139]
Hi, thank you for providing the code used in the paper!

I'm currently trying to train a CUT (not FastCUT) model to morph human face images into Anime style images.
I therefore trying to find good hyperparameters (like `crop_size` and `load_size`, and possibly alter the generator architecture). I noticed that in the finetuning phase (epochs 201 - 400) using the standard dataset mode (`unaligned_dataset`) my setting for `load_size` is ignored in favor of `crop_size`.
In the code ([unaligned_dataset.py Lines 61 to 66](https://github.com/taesungp/contrastive-unpaired-translation/blob/master/data/unaligned_dataset.py#L61-L66)) the comment states that this behavior should only affect **FastCUT**, not CUT.

Can someone (possibly one of the authors) clarify wheter the observed behavior is wrong or the comment is wrong?
Thanks in advance!

**Comment by taesungp** [comment]
Hello, thank you for pointing it out. The comment is wrong, and this augmentation should be applied to both CUT and FastCUT. 

The primary method of data augmentation in training CycleGAN was loading images at 286 resolution, and then making a random crop of size 256. At test time, we load images at 256 resolution and process without cropping. Therefore, there is inherent mismatch in the scale of objects between training and testing time. The objects will appear 286/256 times bigger than its actual size. To remove the mismatch, we disable this augmentation for the last 200 epochs when the learning rate decays. 

In particular, while this could be a good data augmentation for in-the-wild datasets like horses and zebras, it might not be a good method for landmark-aligned datasets like faces, because important locations like eyes, nose, and mouth are baked into the coordinates. So perhaps for your application, it may be better to entirely disable this augmentation, by setting both `load_size` and `crop_size` to the same value.

**Comment by taesungp** [comment]
I think keeping `load_size` not too much bigger than `crop_size` is better, because it will provide more context to the generator. If `crop_size` is too small, the cropped images will be zoomed in too much. Having said so, the largest I experimented with `load_size/crop_size` was `1024/384` for GTA2Cityscapes training. It was mostly because the GPU memory was quite limited when I trained it 5 years ago, and I would've used a larger `crop_size` if I could.
